src/main/java/core/mvc/tobe/ControllerResolver.java

```diff
@@ -0,0 +1,41 @@
+package core.mvc.tobe;
+
+import core.annotation.web.RequestMapping;
+import core.mvc.tobe.support.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.LocalVariableTableParameterNameDiscoverer;
+import org.springframework.core.ParameterNameDiscoverer;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+
+public class ControllerResolver {
+    private static final Logger logger = LoggerFactory.getLogger(ControllerResolver.class);
+
+    private static final List<ArgumentResolver> argumentResolvers = asList(
+            new HttpRequestArgumentResolver(),
+            new HttpResponseArgumentResolver(),
+            new RequestParamArgumentResolver(),
+            new PathVariableArgumentResolver(),
+            new ModelArgumentResolver()
+    );
+
+    private static final ParameterNameDiscoverer nameDiscoverer = new LocalVariableTableParameterNameDiscoverer();
+
+    public void addHandlerExecution(Map<HandlerKey, HandlerExecution> handlers, final Object target, Method[] methods) {
+        Arrays.stream(methods)
+                .filter(method -> method.isAnnotationPresent(RequestMapping.class))
+                .forEach(method -> {
```

Comment:
```suggestion
                .forEach(method -> createHandlers(handlers, target, method));
```

이런식으로 메서드로 추출하면 어떨까요?

Reply:
넵 메서드로 추출하여 stream 내부 코드를 정리하고, 메서드 분리를 진행하였습니다!